### PR TITLE
add private key access instructions

### PR DESCRIPTION
**Pull request by camillescott**
instructions for accessing with the private key file through a local terminal

### Things to check off before merging:

- [ ] Files are listed in `toc.rst` so they will show up on the side bar.
- [ ] The tutorial assumes we're starting from a blank `Ubuntu 16.04` image on Jetstream.

### CODE REVIEW

**Comment by camillescott** [comment]
@ctb @ljcohen @luizirber

**Comment by johnsolk** [comment]
#218

File: jetstream/login.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Cryptographic keys are a convenient and secure way to authenticate without having to use
+passwords. They consist of a pair of files called the public and private keys: the public part can
+be shared with whoever you'd like to authenticate with (in our case, Jetstream!), and the private
+part is kept "secret" on your machine. Things that are encrypted with the public key can be be
+decrypted with the private key, but it is computationally intractable (ie, it would take on the
+order of thousands of years) to determine a private key from a public key. You can read more about
+it [here](https://en.wikipedia.org/wiki/Public-key_cryptography).
+
+The good news is that there is already a registered public key for our Jetstream account. However,
+it make use of it, you'll need the private key. And so, we move on!
```
Comment (taylorreiter) [inline jetstream/login.md]: I think this should read "However, to make use of it..."

**Review by taylorreiter — COMMENTED** [review]
One typo, otherwise ready to merge!

**Comment by camillescott** [comment]
ping @taylorreiter

**Review by taylorreiter — APPROVED** [review]

**Pull request merged**